### ISSUE TITLE
Suggestion: Hardcore Integration

### ISSUE THREAD

**Issue by Zeklo** [Ladysnake/Requiem#141]
I'd love to be able to use this mod in Hardcore apart from simply starting in normal mode and working towards an Opus Daemonium to enable a means of infinitely reviving.

It could be through enabling the forced-transformations (ie. dying to zombie turns you into a zombie, dying to lava in the nether turns you into a wither skeleton, etc) while keeping death permanent outside those events, or introducing a one-use item that turns the player into a spirit. Sorta like a totem of undying but not quite.

**Comment by Pyrofab** [comment]
So, with regard to your first suggestion, if you die from a fall, you get bricked, and if you die as a zombie, you get bricked ?

**Comment by Zeklo** [comment]
Yeah, so death by falling would be a permanent death (as it doesn't abide by any transformation conditions).
Death by a zombie would result in becoming a zombie, and any following deaths (by any means) results in permanent death.

The only downside is it would limit the transformations available. Such as Zombie Pigmen & Skeletons, since there are no current "natural" means of becoming them.

**Comment by yuno44907** [comment]
You are boring and...... Dude dont waste time by writing random things

**Comment by Pyrofab** [comment]
Please refrain from posting opinions that do not contribute to the discussion in any productive way.
